zone/lua_parser_events.cpp

```diff
@@ -227,7 +227,7 @@ void handle_npc_waypoint(
 	l_mob_o.push(L);
 	lua_setfield(L, -2, "other");
 
-	lua_pushinteger(L, std::stoi(data));
+	lua_pushinteger(L, Strings::Stoi(data, -1));
```

Comment:
Do we need the `-1` since this is the default value anyway?

Reply:
0 appears to be the default value of a waypoint unless I'm misreading

Reply:
I meant that the default second parameter of `Strings::Stoi` is `-1`.

Reply:
Ahh, good call. I remember making this 0 before. It's already been weeks and forgot what I had done. The fallback should be 0 by default

Reply:
Awesome, sounds good.